src/main/java/net/notfab/lindsey/core/commands/fun/Quote.java

```diff
@@ -0,0 +1,111 @@
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+import static net.notfab.lindsey.framework.command.Modules.FUN;
+
+@Component
+/**
+ * Quote is a simple command which "generate" a random quote every time that is called.
+ */
+public class Quote implements Command {
+
+    private static final String QUOTES_URL = "https://type.fit/api/quotes";
+    private static final OkHttpClient client = new OkHttpClient().newBuilder()
+            .followSslRedirects(true)
+            .build();
+
+    @Autowired
+    private Messenger msg;
+
+    @Override
+    public CommandDescriptor getInfo() {
+        return new CommandDescriptor.Builder()
+                .name("quote")
+                .module(FUN)
+                .permission("commands.quote", "Required permission to use the command")
+                .build();
+    }
+
+    @Override
+    public boolean execute(Member member, TextChannel channel, String[] args, Bundle bundle) {
+        try {
+            Request request = new Request.Builder()
+                    .url(QUOTES_URL)
+                    .addHeader("Accept", "application/json")
+                    .addHeader("Content-Type", "application/json")
+                    .build();
+
+            Response response = client.newCall(request).execute();
+            String strJson = response.body().string();
+            HashMap<String, String> quote = this.getRandomQuote(strJson);
+            this.sendEmbed(quote, channel);
+
+            return true;
+        } catch (IOException e) {
+            this.handleError(channel);
+            return false;
+        }
+    }
+
+    /**
+     * Build quotes array and select randomly a quote
+     * @param json Json on string format that will be parsed into an array in order to select the quote
+     * @return A HashMap containing the quote info, like it's author and the text(phrase)
+     */
+    private HashMap<String, String> getRandomQuote(String json) {
+        JSONArray jsonArray = new JSONArray(json);
+
+        Random rand = new Random();
+        int randomIndex = rand.nextInt(jsonArray.length());
+        JSONObject phrase = jsonArray.getJSONObject(randomIndex);
+
+        HashMap<String, String> quote = new HashMap<>();
+        quote.put("text", phrase.getString("text"));
+        quote.put("author", phrase.getString("author"));
+
+        return quote;
+    }
+
+    /**
+     * Method responsible to build embed that will show the selected random quote.
+     * @param  quote Quote HashMap object that will be used to build embed
+     * @param channel Channel that the embed will be sent
+     */
+    private void sendEmbed(HashMap<String, String> quote, TextChannel channel) {
+        EmbedBuilder embed = new EmbedBuilder();
+
+        embed.setTitle(quote.get("text"));
+        embed.setDescription(quote.get("author"));
+
+        channel.sendTyping().completeAfter(1, TimeUnit.SECONDS);
```

Comment:
Avoid using complete, as it is blocking and can cause several other issues.

No need to send typing animations also.